docs/rules/no-html-safe.md

```diff
@@ -0,0 +1,113 @@
+# no-html-safe
+
+`htmlSafe` marks a string as safe for unescaped output with Ember templates so you can render it as HTML. `htmlSafe` does **not** perform input sanitization. While useful this can inadvertently open you up to Cross-site Scripting (XSS) vulnerabilities, especially if the string was generated from user input or some other untrusted source. **You should only ever use `htmlSafe` with trusted or sanitized input**.
+
+## Rule Details
+
+This rule prevents importing the `htmlSafe` utillity from `@ember/template` (or `@ember/string` for older Ember versions);
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+import { htmlSafe } from '@ember/template';
```

Comment:
I would vaguely prefer to fail at the usage site, not the import site. 🤔

Reply:
I agree with this so I am updating the location where the violation is flagged in a follow-up PR: https://github.com/ember-cli/eslint-plugin-ember/pull/1080